torch/ao/quantization/_quantize_dbr.py

```diff
@@ -104,11 +119,10 @@ def prepare(model, qconfig_dict, example_inputs, inplace=False, allow_list=None,
             child.qconfig = None  # type: ignore[assignment]
         elif isinstance(child, torch.nn.LSTM):
             # TODO: fix LSTM handling in eager mode static quant and remove this
-            child.qconfig = None
+            qconfig_dict['object_type'][torch.nn.LSTM] = None
```

Comment:
qq: are we going to fix this with default custom_module support?

Reply:
The current logic in DBR for quantization of LSTM is just to make models with LSTMs not crash, it doesn't actually support LSTM quantization.  We would need a couple of PRs to add the support to the framework, it should be pretty straightforward.